core/extension/index.d.ts

```diff
@@ -49,11 +49,12 @@ export {
 	MessageBoxType,
 	MessageBoxIconType,
 	MessageBoxValue,
+	SelectionSetEventType,
```

Comment:
is the sorting arbitrary? makes difficult to tell what actually changed vs what moved.

Reply:
agreed, i was initially thinking my local changes were just re-ordering but there were a few new additions to the api. looking into seeing if we can keep it sorted & stable...

Reply:
@johnnyd710 @aruniverse can you guys file a bug to fix the ordering problem?

@aruniverse let's just manually fix things for this release.